lib/moip2/customer_api.rb

```diff
@@ -10,12 +10,35 @@ def base_path
       "/v2/customers"
     end
 
+    def base_path_credit_card
+      "/v2/fundinginstruments"
+    end
+
     def show(customer_external_id)
       Resource::Customer.new client, client.get("#{base_path}/#{customer_external_id}")
     end
 
     def create(customer)
       Resource::Customer.new client, client.post(base_path, customer)
     end
+
+    def add_credit_card(customer_external_id, credit_card)
+      Resource::CreditCard.new client, client.post(
+        "#{base_path}/#{customer_external_id}/fundinginstruments",
+        credit_card,
+      )
+    end
+
+    def delete_credit_card(credit_card_id)
+      code = client.delete("#{base_path_credit_card}/#{credit_card_id}")
+
+      code.between?(200, 299)
+    end
+
+    def delete_credit_card!(credit_card_id)
```

Comment:
Esse método tá estranho...o `!` em Ruby é pra delimitar ações destrutivas. O método de cima vai ser destrutivo de qualquer maneira, pq manda um `DELETE`, pode dar a intuição errada pro usuário.

Também esse método iria retornar `nil` se tivesse  passado `true` lá em cima, mas se tivesse dado qualquer outra coisa tipo `500` ele ia retornar `Credit Card Not Found` o que não é exatamente a mensagem correta, certo?